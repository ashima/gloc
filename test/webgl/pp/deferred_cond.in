#if E
#define X 1
#else
#define X 2
#endif

void main() { X; }
