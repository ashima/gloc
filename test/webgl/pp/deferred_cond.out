File 0, line 1, col 4 - 5:
ambiguous preprocessor conditional branch: E

Fatal: unrecoverable preprocessor divergence (4)
